tests/FakeItEasy.Tests/FakeFacadeTests.cs

```diff
@@ -89,6 +89,18 @@ public void ClearConfiguration_should_be_guarded()
         }
 
         [Fact]
+        public void ClearRecordedCalls_should_be_guarded()
+        {
+            // Arrange
+
+            // Act
+
+            // Assert
+            Expression<System.Action> call = () => this.facade.ClearRecordedCalls(A.Dummy<object>());

```

Comment:
I agree on both points.

However, the code follows along with all the other tests I've seen in which the AAA phases are spelled out in comments.

I don't mind changing, but figure if we do, we might as well sweep all of the instances. Defer?